reactive/kata.py

```diff
@@ -120,3 +121,15 @@ def publish_config():
         name='kata',
         binary_path='/usr/bin/kata-runtime'
     )
+
+
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    """Set status during series upgrade."""
+    status_set('blocked', 'Series upgrade in progress')
+
+
+@hook('post-series-upgrade')
+def post_series_upgrade():
+    """Reset status to active after series upgrade."""
+    status_set('active', 'Kata runtime available')
```

Comment:
This status could get clobbered, too, but I'm not as worried about it since the "Kata runtime available" status has always been vulnerable to being clobbered.